### ISSUE TITLE
Adding Vietnamese

### ISSUE THREAD

**Issue by crystaltype** [itfoundry/Poppins#25]
Hi,

I see that this type doesn't have Vietnamese yet.

I'm willing to contribute the complete Vietnamese character set directly, by sending you a 'vietnamese only' glyphapp file for you to review and integrate.

Or, you can make changes and then push your work to master and I will give further feedback :)

Please respond ASAP. Thanks!

P/S: I'm a Vietnamese native speaker and a self-study graphic designer/type designer. I have been working with fonts for over 5 years! I'm here to (hopefully) make google fonts a better place for people using Vietnamese! :) Thanks!

cc': @davelab6

**Comment by auduongtuan** [comment]
Hope Poppins can support Vietnamese <3

**Comment by dasklney** [comment]
Yes, this is a wonderful font, support Vietnamese will make it more great!

**Comment by ntnhuy** [comment]
do we support Vietnamese? Thanks

**Comment by vnt87** [comment]
I would like to echo this request as well. Adding Vietnamese would be a great addition to this awesome typeface

**Comment by satyarajpurohit** [comment]
If @davelab6 says, we'd be happy to add Vietnamese support, :)

**Comment by JapanYoshi** [comment]
Since this seems not to be getting traction (I am completely for adding Vietnamese support to Poppins, even if it means making a fork of it), in the meantime, may I suggest TeX Gyre Adventor for a replacement font? It's a free copy of Avant Garde Gothic that supports Vietnamese, among lots of TeX features.

**Comment by dasklney** [comment]
@davelab6 how is your comment?

**Comment by lucazsvn** [comment]
I'm still waiting.

**Comment by dinhanhthi** [comment]
Waiting for this!

**Comment by nnaliu** [comment]
Still waiting for this too!

**Comment by dinhanhthi** [comment]
Is there any news?

**Comment by andyngdz** [comment]
Hello, any updates on this. I love to use this font in Vietnamese :pray:

**Comment by shirenez** [comment]
For anyone still looking for the vietnamese poppins font, heres the font
https://drive.google.com/drive/folders/1rnHUoOup9YmZXFR1CwR337sLt7e0PmeS?usp=sharing

**Comment by Didab877** [comment]
> For anyone still looking for the vietnamese poppins font, heres the font https://drive.google.com/drive/folders/1rnHUoOup9YmZXFR1CwR337sLt7e0PmeS?usp=sharing

Thanks for the share but these appears to be the version localized by SVN, they are not freeware so I don't think they are distributable. Also it's only a few weights, not the whole family. Still, we're hoping for official support so we don't have to mess around with custom font embedding.

**Comment by badcaf3** [comment]
2025 and I’m still hoping the Poppins font officially supports Vietnamese!

**Comment by conghuyn** [comment]
Now 2026:D
